tools/plop-templates/component-component.hbs

```diff
@@ -7,7 +7,7 @@ export interface {{pascalCase component-name}}Props extends HTMLPasteProps<"div"
   children?: React.ReactNode;
   /**
    * Overrides the default element name to apply unique styles with the Customization Provider
-   * @default '{constantCase component-name}'
+   * @default '{{constantCase component-name}}'
```

Comment:
question: did you test out these 2 hbs file changes with the script? I was having trouble using double curly brackets since that's the format plop already uses for their own variables

Reply:
I just ran this for a before and after. In the case of the comment you are right the 2 curly braces, it makes it do some function but without both it just printed the text which was not correct. Here are a before and after of how the changes affected the output. Package name used in the script "TestComponent"

| Before   | After |
| -------- | -------- |
| @default '{constantCase component-name}'  |                 @default 'TEST_COMPONENT' |
| export TestComponent from './TestComponent' |     export { TestComponent } from './TestComponent' |